### PR TITLE
observerward 2024.7.9

### PR DESCRIPTION
**Pull request by cn-kali-team**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/o/observerward.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 class Observerward < Formula
-  desc "Cross platform community web fingerprint identification tool"
-  homepage "https://0x727.github.io/ObserverWard/"
-  url "https://github.com/0x727/ObserverWard/archive/refs/tags/v2024.3.22.tar.gz"
-  sha256 "6236a93c4672da1f18d6484a61316665ef125c34219db1845361eb2941c667e1"
-  license "MIT"
+  desc "Web application and service fingerprint identification tool"
+  homepage "https://emo-carb.github.io/observer_ward/"
```
Comment (chenrui333) [inline Formula/o/observerward.rb:3]: this is 404?

```
$ curl -I https://emo-carb.github.io/observer_ward/
HTTP/2 404
server: GitHub.com
```

Comment (cn-kali-team) [reply Formula/o/observerward.rb]: spelling errors in words
https://emo-crab.github.io/observer_ward/

**Comment by chenrui333** [comment]
cargo registry issue

**Review by chenrui333 — APPROVED** [review]

File: Formula/o/observerward.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 class Observerward < Formula
-  desc "Cross platform community web fingerprint identification tool"
-  homepage "https://0x727.github.io/ObserverWard/"
-  url "https://github.com/0x727/ObserverWard/archive/refs/tags/v2024.3.22.tar.gz"
-  sha256 "6236a93c4672da1f18d6484a61316665ef125c34219db1845361eb2941c667e1"
-  license "MIT"
+  desc "Web application and service fingerprint identification tool"
+  homepage "https://emo-crab.github.io/observer_ward/"
+  url "https://github.com/emo-crab/observer_ward/archive/refs/tags/v2024.7.9.tar.gz"
+  sha256 "e0c46e6a10c8ed386ea400a78b458659d39ee2a86a58a73361b4644222a6fc0c"
```
Comment (chenrui333) [inline Formula/o/observerward.rb:5]: looks like the tag got changed?
```suggestion
  sha256 "d41bda274d0572df8062c31effa3f93c35ce7c8409297ee6157b4529de5f6332"
```

Comment (cn-kali-team) [reply Formula/o/observerward.rb]: yes

**Review by branchv — APPROVED** [review]

**Pull request merged**